# in the acf file of Half Life Blue Shift the name
# is under the UserConfig tag.
# so also look there for the name in case it is not found.
# don't know if other games are also affected.
Index: lutris-0.3.8/lutris/util/steam.py
===================================================================
--- lutris-0.3.8.orig/lutris/util/steam.py	2016-08-28 02:01:57.000000000 +0200
+++ lutris-0.3.8/lutris/util/steam.py	2016-08-28 02:04:23.581070498 +0200
@@ -354,8 +354,12 @@
         return self.appmanifest_data.get('AppState') or {}
 
     @property
+    def user_config(self):
+        return self.app_state.get('UserConfig') or {}
+
+    @property
     def name(self):
-        return self.app_state.get('name')
+        return self.app_state.get('name') or self.user_config.get('name')
 
     @property
     def slug(self):
